shared/src/database/services/local-partner/local-partner.service.ts

```diff
@@ -1,9 +1,69 @@
-import { UserRole } from '@prisma/client';
+import { LocalPartner, UserRole } from '@prisma/client';
 import { BaseService } from '../core/base.service';
 import { ServiceResult } from '../core/base.types';
-import { LocalPartnerTableView, LocalPartnerTableViewRow } from './local-partner.types';
+import {
+	LocalPartnerCreateInput,
+	LocalPartnerPayload,
+	LocalPartnerTableView,
+	LocalPartnerTableViewRow,
+	LocalPartnerUpdateInput,
+} from './local-partner.types';
 
 export class LocalPartnerService extends BaseService {
+	async create(localPartner: LocalPartnerCreateInput): Promise<ServiceResult<LocalPartner>> {
+		try {
+			const partner = await this.db.localPartner.create({ data: localPartner });
```

Comment:
We should add a TODO comment here to remind us to implement proper permission checks later.
Right now, the server action could be called directly (e.g. via CURL), so we’ll need to validate the session cookie and user permissions here.